---
content_type: page
title: Syllabus
uid: 212d0bfa-be06-7dd3-562f-fdf45c78f87d
---

Course Meeting Times
--------------------

Lectures: 2 sessions / week, 1.5 hours / session

Course Description
------------------

This course focuses on casting contemporary problems in systems biology and functional genomics in computational terms and providing appropriate tools and methods to solve them. Topics include genome structure and function, transcriptional regulation, and stem cell biology in particular; measurement technologies such as microarrays (expression, protein-DNA interactions, chromatin structure); statistical data analysis, predictive and causal inference, experiment design. The emphasis is on coupling problem structures (biological questions) with appropriate computational approaches.

During the Spring of 2005, Computational Functional Genomics will be taught around an extended case study showing how we can use new high resolution genomic and proteomic data to discover the underlying biological mechanisms that govern transcriptional regulatory programs in yeast and human.

When possible, our case study will focus on the development of stem cells. Contemporary literature and data as well as new directions for research will be discussed. We will explore the principles of analysis at sufficient depth so that students are able to develop new methodologies that are well founded for new biological problems.

Course Outline
--------------

Our case study exploration will be grouped into the following areas:

### How can we use DNA sequence to explain mechanism?

In this module we will examine how we can analyze genome sequences to discover properties that are evident in a single genome (CpG islands), properties that are conserved between genomes (genome structure), and how we can discover DNA sequence elements that implement combinatorial control of gene expression (motif discovery). Lectures 1-4.

### How can we observe the mechanism of transcriptional regulation?

In this module we will examine the application of DNA microarrays for the analysis of gene expression, protein-DNA binding, chromatin structure, chromatin modifying complexes, and RNA polymerase occupancy. Error models and data normalization techniques for high-resolution array technologies will be presented. Using the processed data we will discuss the basis for clustering genes into sets and discovering gene set features that can be used for diagnostic purposes. We will discuss the importance of chromatin structure in contemporary modeling, and review recent research results on the relationship between chromatin structure and transcriptional regulation. Lectures 5-12.

### How can we build predictive network models of transcriptional regulation?

In this module we will build predictive models of transcriptional regulatory networks using probabilistic modeling techniques. We will examine how graphical models can be used to describe key regulatory mechanisms, and use both direct (molecular interaction data) and functional data (expression, phenotype) to constrain the models we learn. We will begin with yeast, and finish this module examining human regulatory networks that are linked to specific diseases. Lectures 12-22.

Team Project
------------

An integral part of the course is a student project component that is based on our case study theme of understanding biological mechanism. We encourage interdisciplinary groups of students to work together to develop novel analysis methodologies to examine recent data. Topics will be chosen by the teams in consultation with us. There will be intermediate (10 minute) and final (20 minute) presentations of each project in class.

Assignments
-----------

Four problem sets will be assigned during the term.

Quizzes
-------

There will be one final quiz at the end of the term.